doc/Migration.md

```diff
@@ -7,6 +7,13 @@ Please see the latest version (`master`) for the most up-to-date information.
 
 ## Guide
 
+### v1.17.0
+
+#### ES2017
+- Theia was updated to ES2017
```

Comment:
minor (`newline`):

```suggestion
#### ES2017

- Theia was updated to ES2017
```

Like @tsmaeder I would agree that maybe we should provide additional or more complete information on how to actually transpile back to es5 if necessary.